ambry-router/src/test/java/com.github.ambry.router/PutManagerTest.java

```diff
@@ -701,7 +760,7 @@ private void setupEncryptionCast(VerifiableProperties vProps) throws GeneralSecu
    * @param shouldCloseRouterAfter whether the router should be closed after the operation.
    */
   private void submitPutsAndAssertSuccess(boolean shouldCloseRouterAfter) throws Exception {
-    submitPut().await();
+    submitPut().await(5, TimeUnit.SECONDS);
```

Comment:
Create a constant for the timeout length?

Reply:
Done